src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -155,14 +171,17 @@ public function theShippingMethodIsDisabled(ShippingMethodInterface $shippingMet
      * @param string $locale
      * @param array $configuration
      * @param string $calculator
+     * @param bool $enabled
      */
     private function createShippingMethod(
         $name,
         $code = null,
         ZoneInterface $zone = null,
         $locale = 'en',
         $configuration = ['amount' => 0],
-        $calculator = DefaultCalculators::FLAT_RATE
+        $calculator = DefaultCalculators::FLAT_RATE,
+        $enabled = true,
+        $addForCurrentChannel = true
```

Comment:
We need to create shipping method not assigned to any channel to check if its not available - see [this scenario](https://github.com/Sylius/Sylius/pull/5220/files#diff-35c7c77e6ee2837180922a35f25cf2d0R35);